client/src/components/BoulderCard.jsx

```diff
@@ -102,19 +102,16 @@ const WrapperLong = styled.li`
 `;
 
 const StartPic = styled.img`
-  object-fit: cover;
-  width: 100%;
+  object-fit: contain;
+  max-width: 100%;
```

Comment:
Does this property do anything? You cannot go further than 100%, can you? But I am not sure how object-fit works.

Reply:
I have played around with it many times to make it work the way I intend it. I'm afraid it needs to stay as is.